CMakeLists.txt

```diff
@@ -24,14 +25,18 @@ set(virtualoutput_HEADERS
 	src/v4l2sinkproperties.h)
 
 include_directories(
```

Comment:
I've dropped that completely, current obs don't use this directory anymore and it wasn't used

Also it should be taken into account by findpackage(Obs) already.